components/expenses/ExpenseForm.js

```diff
@@ -570,6 +576,30 @@ const ExpenseFormBody = ({
                     {errors.payoutMethod.data.currency.toString()}
                   </Box>
                 )}
+                <StyledHr flex="1" borderColor="#ffffff" mx={2} />
+                <StyledButton
+                  type="button"
+                  buttonStyle="borderless"
+                  width={['100%', 'auto']}
+                  color="red.500"
+                  mt={1}
+                  mx={[2, 0]}
+                  mr={[null, 3]}
+                  whiteSpace="nowrap"
+                  onClick={() => {
+                    if (window.confirm(formatMessage(msg.confirmResetExpense))) {
```

Comment:
🚫 I am dubious on using this kinda native browser form for the confirmation. My opinion is that here we need to add a modal using probably using `StyledModal.js`. 

cc: @Betree

Reply:
Hmm, we can use `StyledModal`. Should be easy.

Reply:
I was the one suggesting that we use a `confirm` for this in https://github.com/opencollective/opencollective/issues/4625#issuecomment-932205130. I think it's OK to have it for that kind of feature that is not meant to be used very often. We do it in other parts of the code.

But having a custom modal is always nicer, thanks @snitin315, for going the extra mile.